example.json

```diff
@@ -95,6 +95,15 @@ Content Object classes
 "_classes": "hide-page-header",
 "_comment": "Hides the page header on the page",
 
+"_classes": "header-color",
+"_comment": "Sets up the header solid background colour mixin. Combine with one of the following classes e.g. 'header-color background'. Note: the following class must be a predefined variable. Mixin can be extended to include client colours. Can be set for menu and/or page header.",
+"_classes": "transparent-light",
```

Comment:
Very minor point - could a new line be added between each entry to make it a little easier to read please